volatility3/framework/plugins/linux/pslist.py

```diff
@@ -57,7 +78,37 @@ def _generator(self):
             if task.parent:
                 ppid = task.parent.pid
             name = utility.array_to_string(task.comm)
-            yield (0, (pid, ppid, name))
+            uid = task.real_cred.uid.val
+            gid = task.real_cred.gid.val
+
+            # task.(real_)start_time contains amount of nanoseconds since boottime
+            create_timespec = task.real_start_time if hasattr(task, "real_start_time") else task.start_time
+            create_timespec_sec = int(create_timespec / 1e9)
```

Comment:
This looks like a constant of some kind.  If it's used throughout the OS, then probably best to move it to constants.linux so that other plugin are consistent with this one...